File: c/quesn42.c
//Write a program to check if a number entered is an Armstrong Number.

#include<stdio.h>
#include<math.h>

int main()    
{  

int n,x,r,sum=0;    
printf("Enter the number you want to check:");    
scanf("%d",&n);                // taking number from user
x=n;  

while(n>0)    
{    
r=n%10;    
sum=sum+(r*r*r);    
n=n/10;    
}   

if(x==sum)    
printf("Entered number is an armstrong  number ");    
else    
printf("Entered number is not an armstrong number");    

return 0;  
}   

File: python/Quesn42.py
# Vishnu Kumar
# ME20B2001
# Question 42

n=int(input("Enter the number of your choice:\n"))

x=int(n)
sum=int(0)


while(n>0):
    r=int(n%10)
    sum=sum+(r*r*r)
    n=int(n/10)

if x==sum :
    print("Entered number is an armstrong  number ") 
    
else:
    print("Entered number is not an armstrong number")